website/docs/d/resource_list.html.markdown

```diff
@@ -391,6 +426,8 @@ The following arguments are supported:
     * `vcd_network_routed_v2`
     * `vcd_network_isolated_v2`
     * `vcd_nsxt_network_imported`
+    * `vcd_nsxt_alb_service_engine_group`
+    * `vcd_nsxt_alb_edgegateway_service_engine_group`
```

Comment:
This one remains untested and prone to breaking because it needs a lot of prerequisites in this resource list. Maybe we could pull in the data source to relevant ALB test where all the prerequisites are created anyway.

Reply:
Modified `TestAccVcdNsxtEdgeGatewayServiceEngineGroupDedicated` so it uses also `vcd_resource_list` data source.
Acceptance and binary test passed